storage/time-series/influxdb/retriever/app/express/routes/v1/Devices.js

```diff
@@ -43,21 +48,22 @@ module.exports = ({ mountPoint, queryDataByField, queryDataByMeasurement }) => {
             logger.debug(`device-route.get: req.params=${util.inspect(req.params)}`);
             logger.debug(`device-route.get: req.query=${util.inspect(req.query)}`);
 
-
             try {
               const { deviceId } = req.params;
+              const { accept } = req.headers;
```

Comment:
Ao invés de recuperar o `accept` do header e aplicar um regex. Acho que seria mais simples e confiável usar `req.accepts(['json', 'csv'])`. Veja http://expressjs.com/en/5x/api.html#req.accepts.

Também seria importante retornar um erro `406 "Not Acceptable` caso o formato solicitado não seja suportado.